examples/service-fastapi/app/__init__.py

```diff
@@ -1 +1 @@
-from .app import app
+from .app import app  # noqa
```

Comment:
```suggestion
from .app import app

__all__ = ["app"]
```
The PEP8  suggestion is to use `__all__`
https://peps.python.org/pep-0008/#public-and-internal-interfaces

Reply:
I've not been so keen on `__all__` because it require us to use more code in these situations, even though just having two variables makes it something of little importance. I figure the key benefit of `__all__` is when you have things declared in `__init__.py` that isn't supposed to be exported, and you can omit them in an `__all__` list.

We have three `from ... import *` statements in two files, something the PEP also suggest we avoid. The `autoflake` autoformatter can do something related to star imports, documented like this:

```
  --expand-star-imports
                        expand wildcard star imports with undefined names; this only triggers if there is only one star import in the file; this is skipped if there are any uses of `__all__` or `del` in the file
```

Ah I figure we shouldn't overthink this detail though, so lets go with what you suggested - either way is fine as long as we don't have massive lists that are manually maintained in my mind!

Reply:
I did originally add a comment along the lines of "should we have some team code style guide for the use of `import *`", then decided this wasn't the time to start that discussion :smile: